### ISSUE TITLE
Documentation: Charts can be displayed as Heatmaps

### ISSUE THREAD

**Issue by kickoke** [netdata/learn#937]
TL;DR: The Netdata cloud engineering team works on a new way to visualize data: Heatmaps!

See [issue 265](https://github.com/netdata/netdata-cloud/issues/265). 

This initiative is driven by @hugovalente-pm and is the point of contact for further research.

## Research: 
- [ ] Existing documents: Collect input from engineers

## Production

- [ ] Existing documents: Update collectors that support heatmaps
- [ ] Existing document: Update section to describe how to change chart types. https://learn.netdata.cloud/docs/cloud/visualize/interact-new-charts
- [ ] Existing document: API documentation on swagger should contain the option "heatmap" (Done by cloud BE team)
- [ ] Get technical review
- [ ] Get peer review
- [ ] Publicize new feature @jlbriston 

## Publish

- [ ] Merge PR
- [ ] Approve ingest PR on netdata/learn
- [ ] Publish social media post for heatmaps
       @jlbriston You decide where to make this feature known the best. :)
